pkg/machinery/config/encoder/documentation.go

```diff
@@ -74,8 +75,28 @@ func (d *Doc) Describe(field string, short bool) string {
 
 // Example represents one example snippet for a type.
 type Example struct {
-	Name  string
-	Value interface{}
+	sync.Once
+	valueMutex sync.RWMutex
```

Comment:
```suggestion

	valueMutex sync.RWMutex
```

https://dmitri.shuralyov.com/idiomatic-go#mutex-hat